### PR TITLE
Fix numbering of Fastly TLS certificate steps

### PR DESCRIPTION
**Pull request by brucebolt**
The indentation was not correctly aligned, resulting in the numbering starting again from 1, even though the sections were explicitly numbered.

### CODE REVIEW

File: source/manual/request-fastly-tls-certificate.html.md [hunk 1]
```diff
@@ -11,61 +11,61 @@ When transitioning HTTPS domains, these are the steps to request a TLS certifica
...
+    - redirector-cdn-ssl-businesslink.production.govuk.service.gov.uk
+    - redirector-cdn-ssl-directgov.production.govuk.service.gov.uk
+    - redirector-cdn-ssl-events-businesslink.production.govuk.service.gov.uk
+    - www-cdn.production.govuk.service.gov.uk
+    - www-gov-uk.map.fastly.net
+    - 151.101.0.144, 151.101.64.144, 151.101.128.144, 151.101.192.144
+    ```
 
-   ![Photo of the step 4](images/fastly/4.png)
+    ![Screnshoot of the Fastly website user interface on the "TLS subscriptions" tab, nested under "TLS management". A form is shown where multiple domains can be added into a text area.](images/fastly/4.png)
```
Comment (ChrisBAshton) [inline source/manual/request-fastly-tls-certificate.html.md:59]:
```suggestion
    ![Screnshot of the Fastly website user interface on the "TLS subscriptions" tab, nested under "TLS management". A form is shown where multiple domains can be added into a text area.](images/fastly/4.png)
```

**Review by ChrisBAshton — APPROVED** [review]
Awesome job ⭐

File: source/manual/request-fastly-tls-certificate.html.md (hunk 1 above)
Comment (brucebolt) [reply source/manual/request-fastly-tls-certificate.html.md]: Oops!  The screenshot version of a photoshoot?

Comment (ChrisBAshton) [reply source/manual/request-fastly-tls-certificate.html.md]: oops my correction has a typo too!

**Pull request merged**